Libraries/PeriphDrivers/Source/I2C/i2c_reva.c

```diff
@@ -1440,11 +1440,7 @@ void MXC_I2C_RevA_MasterAsyncHandler(int i2cNum)
         }
 
         /* Call the callback */
-        if (i2c->intfl0 & MXC_I2C_REVA_ERROR) {
```

Comment:
Why would we want to always pass E_COMM_ERR? I'm confused

Reply:
Whoops, cut the wrong line.  https://github.com/Analog-Devices-MSDK/msdk/pull/781/commits/e492743623b0de3fbc59fa7a0d9d62d938b0d67d